examples/osp-sles.yaml

```diff
@@ -0,0 +1,455 @@
+# Copyright 2021 The Operating System Manager contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: operatingsystemmanager.k8c.io/v1alpha1
+kind: OperatingSystemProfile
+metadata:
+  name: osp-sles
+  namespace: cloud-init-settings
+spec:
+  osName: sles
+  osVersion: "20.04"
```

Comment:
That's wrong, actually the version that we tested machine controller with(IIRC) was SLES 15 SP1, so probably should be `osVersion: "15-sp-1"